keras/export/export_lib_test.py

```diff
@@ -611,7 +631,7 @@ def test_export_archive_errors(self):
         ):
             export_archive.add_endpoint(
                 "call",
-                model.call,
+                model.__call__,
```

Comment:
Any issue with `call`?

Reply:
No, both work. But I thought you're *not* supposed the `call` directly just in general (and `model.export` uses `__call__`).